src/org/ods/orchestration/usecase/JiraUseCase.groovy

```diff
@@ -256,13 +256,20 @@ class JiraUseCase {
     }
 
     void matchTestIssuesAgainstTestResults(List testIssues, Map testResults, Closure matchedHandler, Closure unmatchedHandler = null) {
+        def duplicateKeysErrorMessage = "Error: found duplicated Jira tests. Check tests with key: "
```

Comment:
@angelmp01 @s2oBCN @jorge-romero this error message is not at all clear to the end-user. Can you please be more specific?

Reply:
@angelmp01 What about... "Error: the following test cases are implemented multiple times each: JIRA-1, JIRA-2." We should as well omit the [ ] in the output. Thanks.